internal/mode/static/manager.go

```diff
@@ -199,41 +210,58 @@ func StartManager(cfg config.Config) error {
 		firstBatchPreparer,
 	)
 
-	if err = mgr.Add(eventLoop); err != nil {
+	if err = mgr.Add(&leaderOrNonLeaderRunnable{eventLoop}); err != nil {
```

Comment:
Why is this change needed?

Reply:
if leader election is enabled in the Manager, to run a Runnable when leader election is not enabled (which is needed for the eventLoop), you need to explicitly mark it to not require leader election. That's accomplished by implementing manager.LeaderElectorRunnable with 
```
func (r *leaderOrNonLeaderRunnable) NeedLeaderElection() bool {
	return false
}
```